lib/app-components/addon/components/project-contributors/list/template.hbs

```diff
@@ -1,28 +1,33 @@
-<div class='row hidden-xs'>
-    <div class='col-sm-3 col-sm-offset-2'>
-        <strong>{{t 'app_components.project_contributors.list.name'}}</strong>
-    </div>
-    <div class='col-sm-3'>
-        <strong>{{t 'app_components.project_contributors.list.permissions'}}</strong>
-        <span>
+{{#if this.isNotMobile}}
```

Comment:
You could do `{{#unless this.isMobile}}` here instead. Then you wouldn't need the extra getter.

Reply:
If you were doing anything but a simple `#if`, then you would do `(not this.isMobile)`, but if has a special thing where they would rather do an `#unless` rather than an `#if (not…`.